river/Config.zig

```diff
@@ -85,6 +90,18 @@ repeat_rate: u31 = 25,
 /// Keyboard repeat delay in milliseconds
 repeat_delay: u31 = 600,
 
+/// Cursor hide timeout in milliseconds
+cursor_hide_timeout: u31 = 0,
+
+/// How many pixels the cursor has to be moved to reset the hide tiemout.
+cursor_hide_jitter: u32 = 0,
+
+/// How many pixels the cursor has to be moved to unhide it.
+cursor_unhide_jitter: u32 = 0,
```

Comment:
Looks like these should have been removed in 21e418f6fbd0b0ec5ef91a5f7d0a6a93dcad3060